### ISSUE TITLE
why matterbridge delete empty line of message from telegram?

### ISSUE THREAD

**Issue by liqsliu** [42wim/matterbridge#1561]
**Describe the bug**
When there is empty line in message from telegram, the empty line will be deleted by matterbridge

**To Reproduce**
in telegram, send message: "1
2

3",
in matrix, get "1
2
3".

**Expected behavior**
in matrix, get message: "1
2

3"

**Screenshots/debug logs**
the official matterbridge group also has the bug.
![Screenshot_20210801-090425_Telegram](https://user-images.githubusercontent.com/5843584/127756025-84cc7bee-5af3-4940-b6df-475cb9289707.png)

**Environment (please complete the following information):**
 

**Additional context**

**Comment by 42wim** [comment]
Correct, empty messages from telegram are ignored.
What's the use-case for empty messages?

**Comment by liqsliu** [comment]
> Correct, empty messages from telegram are ignored.
> What's the use-case for empty messages?

it's not empty message, it 's empty line. 
the empty line of message was deleted.

**Comment by rgegvre** [comment]
I got this too, it's so annoying

**Comment by rgegvre** [comment]
@42wim really need to be fix, and thanks for the most amazing bridge

**Comment by 42wim** [comment]
debug logs please

**Comment by liqsliu** [comment]
> debug logs please

![Screenshot_20220207-122059](https://user-images.githubusercontent.com/5843584/152724065-d6955042-3e36-4521-9946-5ef43c4dac4f.jpg)

I have tested it in your group, and the same problem occurs

**Comment by rgegvre** [comment]
@42wim here is mine
my message in telegram was
1

2

3
4

but in element was shown
1
2
3
4

my debug
```
C:\Users\Administrator\Desktop\bridge>matterbridge.exe -debug  -conf element.tom
l
[0000]  INFO main:         [setupLogger:matterbridge.go:104] Enabling debug logg
ing.
[0000]  INFO main:         [main:matterbridge.go:44] Running version 1.24.0 65c7
ac80
[0000]  INFO router:       [Start:gateway/router.go:66] Parsing gateway gateway1

[0000]  INFO router:       [Start:gateway/router.go:66] Parsing gateway gateway2

[0000]  INFO router:       [Start:gateway/router.go:75] Starting bridge: telegra
m.xxxxx
[0000]  INFO telegram:     [Connect:bridge/telegram/telegram.go:44] Connecting
[0000]  INFO telegram:     [Connect:bridge/telegram/telegram.go:53] Connection s
ucceeded
[0000]  INFO telegram:     [joinChannels:bridge/bridge.go:77] telegram.xxxxx: j
oining -100123456789 (ID: -100123456789telegram.xxxxx)
[0000]  INFO router:       [Start:gateway/router.go:75] Starting bridge: matrix.
xxxxx
[0000]  INFO matrix:       [Connect:bridge/matrix/matrix.go:92] Connecting https
://xxxxxxx.com
[0001]  INFO matrix:       [Connect:bridge/matrix/matrix.go:118] Connection succ
eeded
[0001]  INFO matrix:       [joinChannels:bridge/bridge.go:77] matrix.xxxxx: joi
ning !ldOxqEbdfRGCRhnCRs:xxxxxxx.com (ID: !ldOxqEbdfRGCRhnCRs:xxxxxxx.commat
rix.xxxxx)
[0001]  INFO router:       [Start:gateway/router.go:75] Starting bridge: telegra
m.xxxxx
[0001]  INFO telegram:     [Connect:bridge/telegram/telegram.go:44] Connecting
[0002]  INFO telegram:     [Connect:bridge/telegram/telegram.go:53] Connection s
ucceeded
[0002]  INFO telegram:     [joinChannels:bridge/bridge.go:77] telegram.xxxxx: j
oining -100123456789 (ID: -100123456789telegram.xxxxx)
[0002]  INFO router:       [Start:gateway/router.go:75] Starting bridge: matrix.
xxxxx
[0002]  INFO matrix:       [Connect:bridge/matrix/matrix.go:92] Connecting https
://xxxxxxx.com
[0002]  INFO matrix:       [Connect:bridge/matrix/matrix.go:118] Connection succ
eeded
[0002]  INFO matrix:       [joinChannels:bridge/bridge.go:77] matrix.xxxxx: joi
ning !wtRcmFjObxDBBJkFqE:xxxxxxx.com (ID: !wtRcmFjObxDBBJkFqE:xxxxxxx.commat
rix.xxxxx)
[0002]  INFO main:         [main:matterbridge.go:68] Gateway(s) started succesfu
lly. Now relaying messages
[0015] DEBUG telegram:     [handleRecv:bridge/telegram/handlers.go:154] == Recei
ving event: (*tgbotapi.Message)(nil)
[0015] DEBUG telegram:     [handleRecv:bridge/telegram/handlers.go:210] <= Sendi
ng message from unknown on telegram.xxxxx to gateway
[0015] DEBUG telegram:     [handleRecv:bridge/telegram/handlers.go:211] <= Messa
ge is config.Message{Text:"1\n2\n3\n4", Channel:"-100123456789", Username:"unkn
own", UserID:"-100123456789", Avatar:"", Account:"telegram.xxxxx", Event:"", P
rotocol:"", Gateway:"", ParentID:"", Timestamp:time.Date(1, time.January, 1, 0,
0, 0, 0, time.UTC), ID:"421", Extra:map[string][]interface {}{}}
[0015] DEBUG gateway:      [SendMessage:gateway/gateway.go:482] => Sending confi
g.Message{Text:"1\n2\n3\n4", Channel:"-100123456789", Username:"unknown", UserI
D:"-100123456789", Avatar:"", Account:"telegram.xxxxx", Event:"", Protocol:"te
legram", Gateway:"gateway2", ParentID:"", Timestamp:time.Date(2022, time.Februar
y, 13, 12, 3, 7, 768195600, time.Local), ID:"421", Extra:map[string][]interface
{}{}} from telegram.xxxxx (-100123456789) to matrix.xxxxx (!mFjDwtRcBBJkFqEOb
x:xxxxxxx.com)
[0015] DEBUG matrix:       [Send:bridge/matrix/matrix.go:144] => Receiving confi
g.Message{Text:"1\n2\n3\n4", Channel:"!wtRcmFjObxDBBJkFqE:xxxxxxx.com", Userna
me:"", UserID:"-100123456789", Avatar:"", Account:"telegram.xxxxx", Event:"",
Protocol:"telegram", Gateway:"gateway2", ParentID:"", Timestamp:time.Date(2022,
time.February, 13, 12, 3, 7, 768195600, time.Local), ID:"", Extra:map[string][]i
nterface {}{}}
[0015] DEBUG matrix:       [Send:bridge/matrix/matrix.go:147] Channel !mFjDwtRcB
BJkFqEObx:xxxxxxx.com maps to channel id !wtRcmFjObxDBBJkFqE:xxxxxxx.com
[0015] DEBUG gateway:      [SendMessage:gateway/gateway.go:501] mID matrix.nashi
r: $JnuN8GB9UvGyuMHiDCnSXVClJ_su38_fyz4z71pdVu4
[0015] DEBUG gateway:      [func1:gateway/gateway.go:491] => Send from telegram.
xxxxx (-100123456789) to matrix.xxxxx (!wtRcmFjObxDBBJkFqE:xxxxxxx.com) too
k 373.078ms
[0015] DEBUG matrix:       [handleEvent:bridge/matrix/matrix.go:457] == Receivin
g event: &gomatrix.Event{StateKey:(*string)(nil), Sender:"@telebot:xxxxxxx.com
", Type:"m.room.message", Timestamp:1644782827299, ID:"$JnuN8GB9UvGyuMHiDCnSXVCl
J_su38_fyz4z71pdVu4", RoomID:"!wtRcmFjObxDBBJkFqE:xxxxxxx.com", Redacts:"", Un
signed:map[string]interface {}{"age":88, "transaction_id":"go1644782587771193600
"}, Content:map[string]interface {}{"body":"1\n2\n3\n4", "msgtype":"m.text"}, Pr
evContent:map[string]interface {}(nil)}
```

**Comment by reckel-jm** [comment]
I can confirm the same problem in my groups.

**Comment by reckel-jm** [comment]
Update: I potentially found the problem. It could be line `204` of `bridge/telegram/handlers.go`:

```
rmsg.Text = helper.RemoveEmptyNewLines(rmsg.Text)
```

Maybe we can comment out that line and try whether the problem is solved then (I can't test it at the moment).

**Comment by reckel-jm** [comment]
I have implemented that no in a pull request #1749. With that, the problem should be fixed. In addition, Telegram formattings are now forwarded correctly.
